email_mailbox/src/actions/labels.js

```diff
@@ -158,6 +160,8 @@ export const updateBadgeLabels = labelIds => {
         })
       );
       dispatch(updateBadgeLabelsSuccess(labels));
+      const accounts = await defineBadgeAccounts();
+      dispatch(updateAccounts(accounts));
```

Comment:
Avoid using a different structure action/reducer from the one controlled in the file. Move this to a Account actions and make sure to call it when calling this action as well